hadoop-ozone/ozone-manager/pom.xml

```diff
@@ -144,6 +144,24 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <version>0.9.11</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.ranger</groupId>
+      <artifactId>ranger-intg</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+      <scope>compile</scope>
+      <!-- Workaround to prevent slf4j binding conflicts until ranger-intg is
+       fixed to not introduce this to the classpath -->
+      <exclusions>
+        <exclusion>
+          <groupId>ch.qos.logback</groupId>
+          <artifactId>logback-classic</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.sun.jersey</groupId>
+          <artifactId>jersey-bundle</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Are `elasticsearch` and `lucene` required, which seems to be introduced as transitive dependencies? If not we could exclude them as well?